cmd/oas-expand/expand.go

```diff
@@ -0,0 +1,41 @@
+// CLI utility that expands OAS file to reduce init time.
+package main
+
+import (
+	"flag"
+	"fmt"
+	"os"
+
+	"github.com/tetafro/oas2"
+)
+
+const help = `
+Usage:
+	expand [SPEC_FILE] [CACHE_DIR]
```

Comment:
`oas-expand`?

Reply:
Also, I still suggest to use flag argument for CACHE_DIR. Lets make `[SPEC_FILE]` an argument, and all other options flags. For example, by default we can output expanded spec to stdout, and using a flag like `--target-dir` we can write expanded spec fo a file. Later we can introduce additional flags, like `--target-file`, if we want to, and not break backwards-compatibility.

Reply:
fixed